aiida/restapi/api.py

```diff
@@ -336,3 +335,4 @@ def handle_error(self, e):
 
         else:
             raise e
+    """
```

Comment:
If you want to comment the function, please use # (don't make it a string).
Also, is there a better alternative to commenting it? Do we want to get rid of it or not?